src/seat.c

```diff
@@ -189,8 +189,10 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 		wlr_log(WLR_INFO, "dwt configured");
 		libinput_device_config_dwt_set_enabled(libinput_dev, dc->dwt);
 	}
-	if ((libinput_device_config_click_get_methods(libinput_dev)
-				& dc->click_method) == 0
+
+	if ((dc->click_method != LIBINPUT_CONFIG_CLICK_METHOD_NONE
+			&& (libinput_device_config_click_get_methods(libinput_dev)
+				& dc->click_method) == 0)
```

Comment:
```suggestion
	if ((libinput_device_config_click_get_methods(libinput_dev)
				& dc->click_method) != dc->click_method)
```

Reply:
I like this, but this results unfortunately in comparison with different signs :(

Reply:
As discussed via IRC: refactor intended for the whole thing to use `UINT32_MAX` as "not-configured" placeholder instead.